client/src/index.js

```diff
@@ -14,6 +14,7 @@ import * as serviceWorker from "./serviceWorker";
 import rootReducer from "./reducers";
 
 const store = createStore(rootReducer, applyMiddleware(thunk));
+console.log("redux state: ", store.getState());
```

Comment:
forgot log?

Reply:
Left it there on purpose to temporary replace it with redux-logger.. but I can remove it for now.

Reply:
ah ok, i'm used to just check with redux-devtools if needed, i guess adding a logger could also work